### DISCUSSION TITLE
HW 1 Part 2 Q.2

### DISCUSSION THREAD

**Discussion by UniquelyAbbi** [cooperrc/computational-mechanics#25] [Q&A]
I think I am doing something very wrong. Can someone explain.[
![image](https://user-images.githubusercontent.com/106557922/171739472-5b24ba87-aeb9-4274-b238-7979bf4930b0.png)

**Comment by feliciones** [comment]
From what I gathered, you create a list made of random numbers from 0 to 9, then you append those random indices into a res_lst. Once those indices have been appended, you can add a line outside the for-loop like, `A_99 = res_lst[0] * res_lst[1]`. This line will multiply the first index in the array by the second in the array. This was the route that I took.

**Reply by UniquelyAbbi** [reply]
I did try something similar to what you said but I still get errors
![part 2 #2 1](https://user-images.githubusercontent.com/106557922/171742479-c2d60c25-3767-4eb7-8cac-a3d3ce4f46d3.png)

![image](https://user-images.githubusercontent.com/106557922/171742397-0f39f91c-0727-4341-bfa5-585f0f4c4c76.png)

**Reply by feliciones** [reply]
In my for-loop, the two lines saying `res_lst.append(lst_1[i])` and `res_lst.append(lst_2[i])` are there to append the array and put the first random integer in the first spot of the array and the second random integer in the second spot of the array. This then allows you to call the first index and second index of the array, e.g. `res_lst[0] = 2`, and `res_lst[1] = 5`, so you do `product=res_lst[0]*res_lst[1]` which is equal to calling the 2 and 5 and multiplying them to get 10, and then setting that value to "product".

Please forgive my vocab if I happen to call something the wrong thing.

![image](https://user-images.githubusercontent.com/106564968/171743818-ae83005e-e2f3-42eb-ae53-49c3d0838b74.png)

**Reply by UniquelyAbbi** [reply]
Thank you ill try working on it now. No need to worry about vocab I still appreciate your help.

**Reply by cooperrc** [reply]
> From what I gathered, you create a list made of random numbers from 0 to 9, then you append those random indices into a res_lst. Once those indices have been appended, you can add a line outside the for-loop like, `A_99 = res_lst[0] * res_lst[1]`. This line will multiply the first index in the array by the second in the array. This was the route that I took.

No need to use random numbers. You can use a sequence of numbers from 0 to 9 e.g. `[0, 1, 2, 3, 4, 5, 6, 7, 8, 9]`

**Reply by UniquelyAbbi** [reply]
i used np.meshgrid to get everything working and mixed some things.
